gis/text_to_centreline/sql/helper_functions/function-abbr_street.sql

```diff
@@ -41,11 +41,11 @@ _abbrev_street := regexp_REPLACE(_abbrev_street, '(?<=[A-Z][a-z]+ )(Park)(?![a-z
 _abbrev_street := regexp_REPLACE(_abbrev_street, 'Gate', 'Gt', 'g');
 _abbrev_street := regexp_REPLACE(_abbrev_street, 'Pathway', 'Pthwy', 'g');
 _abbrev_street := regexp_REPLACE(_abbrev_street, '\(.*\)', '', 'g');
-_abbrev_street := regexp_REPLACE(_abbrev_street, 'approximately', '', 'g');
+_abbrev_street := regexp_REPLACE(_abbrev_street, 'approximately', '', 'gi');
 _abbrev_street := regexp_REPLACE(_abbrev_street, ' Grove', ' Grv', 'g');
 _abbrev_street := regexp_REPLACE(_abbrev_street, ' Grv Rd', ' Grove Rd', 'g');
 _abbrev_street := regexp_REPLACE(_abbrev_street, ' Grv Ave', ' Grove Ave', 'g');
 _abbrev_street := regexp_REPLACE(_abbrev_street, 'Rd Cres', 'Road Cres', 'g');
 return _abbrev_street;
 end;
-$BODY$;
+$BODY$;
```

Comment:
there is one permission difference between this and the old function, not sure what this is but we should add it as well
```
GRANT EXECUTE ON FUNCTION gis.abbr_street_stop_using_this(text) TO vz_readonly;
```